File: Model/Proyecto.py
from pydantic import BaseModel
from datetime import date

class ProyectoBase(BaseModel): 
    Proyecto_ID :int 
    Nombre_Proyecto:str
    Descripción:str
    Fecha_Inicio:date
    Fecha_Final:date
    Estado_Proyecto: str # por hacer, en progreso, completado, etc.
        

File: FrondEnd/interno/Models/Proyecto.go
package Models

type Proyecto struct {
	Nombre_Proyecto string `json:"Nombre_Proyecto"`
	Descripción     string `json:"Descripción"`
	Fecha_Inicio    string `json:"Fecha_Inicio"`
	Fecha_Final     string `json:"Fecha_Final"`
	Estado_Proyecto string `json:"Estado_Proyecto"`
}
